### PR TITLE
Update README.md about community meetings

### PR DESCRIPTION
**Pull request by kfaseela**
Now that we are restarting our community meetings, clarify a bit about the process to submit the talks. People are directly reaching out to different members currently, asking how to be involved in community meetings.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +49,26 @@ You can also find them on [the community calendar](https://calendar.google.com/c
 
 Meeting agendas and notes can be accessed in the [working doc](http://bit.ly/istiocommunitymeet).
 
+Interested in presenting? Please send your abstract to istio-steering@googlegroups.com.
+This is the information we need from you:
+    * Name
+    * Role and company
```
Comment (linsun) [inline README.md:55]: Can we add a speaker BIO?

**Review by linsun — COMMENTED** [review]
LGTM overall, left small comment

**Comment by justinpettit** [comment]
LGTM.  I agree with Lin and added another small comment.  Thanks!

File: README.md [hunk 2]
```diff
@@ -49,6 +49,26 @@ You can also find them on [the community calendar](https://calendar.google.com/c
...
+    * Talk title and a short abstract
```
[the remark this answers is not in the document]
Comment (kfaseela) [reply README.md:57]: Added * Talk Format(Presentation/Demo) and Session Length

File: README.md (hunk 1 above)
Comment (kfaseela) [reply README.md]: Done

File: README.md [hunk 3]
```diff
@@ -49,6 +49,28 @@ You can also find them on [the community calendar](https://calendar.google.com/c
...
+    * Talk Format(Presentation/Demo) and Session Length
+    * Email to connect
+
+Suitable content:
+
+    * Istio new feature releases and project updates
+    * Tutorials and demos
+    * Use cases
+    * Content that relates vendor or platform to Istio installation and use
```
Comment (hzxuzhonghu) [inline README.md:66]: It should be ok as long as it is open sourced, otherwise vendor private should not be encouraged or allowed

**Comment by chaltenio** [comment]
I'd suggest creating and publishing a form where folks can submit sessions. This form would help standardize, centralize and organize proposed sessions, in addition to other benefits. 

Happy to open a new issue with more details if that makes sense?

**Comment by kfaseela** [comment]
> I'd suggest creating and publishing a form where folks can submit sessions. This form would help standardize, centralize and organize proposed sessions, in addition to other benefits.
> 
> Happy to open a new issue with more details if that makes sense?

We can move to forms in future when we have considerably large number of submissions.

File: README.md (hunk 3 above)
Comment (kfaseela) [reply README.md:66]: That has been added as a point in the "Unsuitable Content" section

Comment (linsun) [inline README.md:66]:
```suggestion
    * Open source contents that relates vendor or platform to Istio installation and use
```

Comment (linsun) [reply README.md:66]: I suggested a small change to make it clear

**Review by linsun — COMMENTED** [review]
LGTM, i think it is ready to be merged after this small change.

**Comment by linsun** [comment]
> > I'd suggest creating and publishing a form where folks can submit sessions. This form would help standardize, centralize and organize proposed sessions, in addition to other benefits.
> > Happy to open a new issue with more details if that makes sense?
> 
> We can move to forms in future when we have considerably large number of submissions.

Agreed, we can iterate on this.

File: README.md [hunk 4]
```diff
@@ -49,6 +49,28 @@ You can also find them on [the community calendar](https://calendar.google.com/c
...
+    * Talk title and a short abstract
+    * Talk Format(Presentation/Demo) and Session Length
+    * Email to connect
+
+Suitable content:
+
+    * Istio new feature releases and project updates
+    * Tutorials and demos
+    * Use cases
+    * Open source contents that relates vendor or platform to Istio installation and use
```
Comment (linsun) [inline README.md:66]:
```suggestion
    * Open source content that relates vendor or platform to Istio installation and use
```

**Review by linsun — APPROVED** [review]
all comments have been resolved, LGTM

**Pull request merged**